.gitignore

```diff
@@ -7,3 +7,4 @@
 /phpunit.xml
 /tests/database.sqlite*
 /vendor
+/.cursor
```

Comment:
Would it be okay if you added this to your global gitignore, instead of to the project's gitignore?

I know the framework has started accepting these, but I don't know if we are ready to fill ours with them.